src/validation.h

```diff
@@ -866,6 +872,15 @@ class ChainstateManager
     CBlockIndex* getSnapshotBaseBlock() EXCLUSIVE_LOCKS_REQUIRED(::cs_main);
     std::optional<int> getSnapshotHeight() EXCLUSIVE_LOCKS_REQUIRED(::cs_main);
 
+    //! Return true if a chainstate is considered usable.
+    //!
+    //! This might be false for, say, a background validation chainstate which has
```

Comment:
In commit "validation: add CChainState::m_stop_use and ChainMan::isUsable" (c703836160f3eca8073c22a1531bb98ad9666999)

Would change "This might be false, for, say" to "This will be false for". IMO wishy washy comments are hard to understand and don't provide any real future proofing (A veritably false comment is still less confusing than a technically true but mostly misleading comment)

Reply:
Wishywashy comment no more.